### PR TITLE
Update Laravel Version

### PR DESCRIPTION
**Pull request by MarcyLina**
In this PR I update the Laravel version from 5.4 to 5.5.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -7,8 +7,8 @@ APP_URL=http://dev-battle-landing.dev/
 
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
-DB_PORT=3306
-DB_DATABASE=dev-battle-landing
+DB_PORT=3307
```
Comment (mattstauffer) [inline .env.example:10]: Let's keep this at `3306`, since it's only `3307` on your local. Thanks!

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Installation
```
Comment (mattstauffer) [inline readme.md:1]: Lovely! Thanks for adding this in! Note: It still has some of the `confomo` entries in it.

File: tests/Feature/VoteTest.php [hunk 3]
```diff
@@ -15,9 +13,11 @@ class VoteTest extends TestCase
...
-        $response = $this->post('/api/vote', ['framework' => 'react']);
+        $count = Vote::count();
+        $response = $this->post('/api/vote', ['framework' => 'cfhgaydfreact']);
 
-        $this->assertTrue($response->getStatusCode() >= 300);
+        $this->assertEquals(422, $response->getStatusCode());
+        $this->assertEquals(Vote::count(), $count);
     }
 
     /** @test */
```
Comment (mattstauffer) [inline tests/Feature/VoteTest.php:23]: What's this `typescript` file below?

**Review by mattstauffer — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (MarcyLina) [reply .env.example]: Updated!

File: readme.md (hunk 2 above)
Comment (MarcyLina) [reply readme.md:1]: Oopsie! Updated!

File: tests/Feature/VoteTest.php (hunk 3 above)
Comment (MarcyLina) [reply tests/Feature/VoteTest.php:23]: I have no idea. It's gone now!

**Pull request merged**